src/components/docs/Buckets/TagBucket/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import Bucket from '../Bucket';
+
+import s from './styles.module.css';
+
+const TagBucket = ({ icon, tags, title, to }) => (
+  <Bucket className={s.root} to={to}>
```

Comment:
We need to add all aria tags to this and make sure we think about ada throughout all development.

Reply:
@jjwill the bucket component is already an a tag. What would need to be done regarding aria in this case?

Reply:
I'm also not seeing an aria label here: https://stripe.com/docs/payments